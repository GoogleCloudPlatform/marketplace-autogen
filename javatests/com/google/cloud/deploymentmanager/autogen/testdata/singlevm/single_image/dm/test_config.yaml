imports:
- path: wordpress.jinja

resources:
- name: wordpress
  type: wordpress.jinja
  properties:
    zone: us-central1-f
