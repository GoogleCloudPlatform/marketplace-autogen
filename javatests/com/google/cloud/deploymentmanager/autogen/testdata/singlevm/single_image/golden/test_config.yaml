imports:
- path: wordpress.jinja

resources:
- name: wordpress
  type: wordpress.jinja
